ggml-sycl.cpp

```diff
@@ -17287,8 +17482,7 @@ GGML_CALL static bool ggml_backend_sycl_supports_op(ggml_backend_t backend, cons
                     return false;
                 }
                 ggml_type a_type = a->type;
-                if (a_type == GGML_TYPE_IQ4_NL || a_type == GGML_TYPE_IQ2_S ||
-                    a_type == GGML_TYPE_IQ4_XS) {
+                if (a_type == GGML_TYPE_IQ4_NL || a_type == GGML_TYPE_IQ4_XS) {
```

Comment:
```suggestion
                if (a_type == GGML_TYPE_IQ4_NL || a_type == GGML_TYPE_IQ4_XS ||
                    a_type == GGML_TYPE_IQ2_XXS || a_type == GGML_TYPE_IQ2_XS) {
```
Can we add these protections so that the mul mat tests skip rather than assert for unsupported quantization types?

Reply:
This would impact the actual cause of the PR, and would bypass all quants as a whole. I think suppressing assert, we can try to run the quants. But I am open to ideas.

Reply:
Yes I added the conditions and  a prev PR also supported iq3xxs and variants.